lib/components/app/responsive-webapp.js

```diff
@@ -241,7 +248,27 @@ class RouterWrapper extends Component {
         </div>
       </ConnectedRouter>
     )
+
+    return (
+      auth0Config
+        ? (
+          <Auth0Provider
+            audience={AUTH0_AUDIENCE}
+            scope={AUTH0_SCOPE}
+            domain={auth0Config.domain}
+            clientId={auth0Config.clientId}
+            redirectUri={URL_ROOT}
+            {...auth0Callbacks}
+          >
+            {router}
+          </Auth0Provider>
+        )
+        : router
+    )
   }
 }
-const mapStateToWrapperProps = (state, ownProps) => ({ routerConfig: state.otp.config.reactRouter })
+const mapStateToWrapperProps = (state, ownProps) => ({
+  persistence: state.otp.config.persistence,
+  routerConfig: state.otp.config.reactRouter
+})
 export default connect(mapStateToWrapperProps)(RouterWrapper)
```

Comment:
The RouterWrapper name should be changed to something like `AuthRouterWrapper`. Open to other suggestions. Also, it could probably use jsdoc (can be kept simple) at this point.

Reply:
`RouterWrapperWithAuth0`